tests/object_model/xml/test_expression.py

```diff
@@ -103,4 +103,4 @@ def test_nested_braces(self):
 
```

Comment:
Rebase issue by the looks of things.

Reply:
That's what caused the double commit problem and me closing this pull request.